presentation/components/src/test/instance-filter-builder/PresentationInstanceFilterBuilder.test.tsx

```diff
@@ -289,22 +302,81 @@ describe("usePresentationInstanceFilteringProps", () => {
     });
 
     it("does not change selected classes when filtering by properties is disabled", () => {
-      const {result} = renderHook(
+      const { result } = renderHook(
         (props: HookProps) => usePresentationInstanceFilteringProps(props.descriptor, props.classHierarchyProvider, props.enableClassFiltering),
-        {initialProps: {...initialProps, classHierarchyProvider, enableClassFiltering: false}});
+        { initialProps: { ...initialProps, classHierarchyProvider, enableClassFiltering: false } });
 
       const property = result.current.properties.find((prop) => prop.displayLabel === concretePropertiesField2.label);
       result.current.onPropertySelected(property!);
       expect(result.current.selectedClasses).to.be.empty;
     });
 
     it("does not change selected classes when 'onPropertySelected' is invoked with invalid property", () => {
-      const {result} = renderHook(
+      const { result } = renderHook(
         (props: HookProps) => usePresentationInstanceFilteringProps(props.descriptor, props.classHierarchyProvider, props.enableClassFiltering),
-        {initialProps: {...initialProps, classHierarchyProvider, enableClassFiltering: true}});
+        { initialProps: { ...initialProps, classHierarchyProvider, enableClassFiltering: true } });
 
-      result.current.onPropertySelected({name: "invalidProp", displayLabel: "InvalidProp", typename: "string"});
+      result.current.onPropertySelected({ name: "invalidProp", displayLabel: "InvalidProp", typename: "string" });
       expect(result.current.selectedClasses).to.be.empty;
     });
   });
 });
+
+describe("PresentationPropertyRenderer", () => {
+  stubRaf();
+  const className = "TestClassName";
+  const schemaName = "TestSchema";
+
+  before(async () => {
+    await NoRenderApp.startup({
+      localization: new EmptyLocalization(),
+    });
+    await UiComponents.initialize(new EmptyLocalization());
+    await Presentation.initialize();
+    Element.prototype.scrollIntoView = sinon.stub();
+  });
+
+  after(async () => {
+    Presentation.terminate();
+    UiComponents.terminate();
+    await IModelApp.shutdown();
+    sinon.restore();
+  });
+
+  it("renders with badge", () => {
+
+    const testPropertyInfo = createTestInstanceFilterPropertyInfo({
+      className: `${schemaName}:${className}`,
+      categoryLabel: "TestCategoryLabel",
+    });
+
+    const { container, getByText } = render(<PresentationInstanceFilterProperty
+      instanceFilterPropertyInfo={testPropertyInfo} />);
+
+    getByText(testPropertyInfo.propertyDescription.displayLabel);
```

Comment:
We use `getByText` when we know the element exists. When checking for existence, we should prefer `queryByText`.